mission/config/functions.hpp

```diff
@@ -323,6 +325,7 @@ class CfgFunctions
 			file = "functions\systems\sites\create\compositions";
 			class sites_create_compositions_aa {};
 			class sites_create_compositions_camp {};
+			class sites_create_compositions_wreck {};
```

Comment:
no compositions used for this one. https://github.com/Bro-Nation/Mike-Force/pull/386/files#r1742442723